app/controllers/api/v3/ecf_partnerships_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Api
+  module V3
+    class ECFPartnershipsController < Api::ApiController
```

Comment:
The pattern we're following for namespacing is:
```suggestion
module Api
  module V3
    module ECF
      class PartnershipsController < Api::ApiController
    end
  end
end
```